server/schedulers/adjacent_region.go

```diff
@@ -46,16 +46,16 @@ func init() {
 		return func(v interface{}) error {
 			conf, ok := v.(*balanceAdjacentRegionConfig)
 			if !ok {
-				return ErrScheduleConfigNotExist
+				return errs.ErrScheduleConfigNotExist
 			}
 			if len(args) == 2 {
 				leaderLimit, err := strconv.ParseUint(args[0], 10, 64)
 				if err != nil {
-					return errors.WithStack(err)
+					return errs.ErrSchedulerConfig.Wrap(err).FastGenByArgs("leader limit")
```

Comment:
Is there any way to use `errs.ZapError`?

Reply:
Maybe it's better, use wrap instead of zap, in the returned result？

Reply:
Oh, got it.